demo/src/app/components/+dropdown/demos/autoclose/autoclose.html

```diff
@@ -0,0 +1,34 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 mb-3">
+    <p>Default behavior is: not to close dropdown out of the dropdown or after pressing ESC </p>
+    <div class="btn-group" dropdown [autoClose]="false">
```

Comment:
autoClose false is definitely not the default behavior. The first dropdown shouldn't use `autoClose` at all, so remove this `[autoClose]="false"`. The second dropdown must have `[autoClose]="false"` and a corresponding description